ios/Classes/UIImage+native_images.h

```diff
@@ -1,24 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-#import <UIKit/UIKit.h>
-
-@interface UIImage (native_images)
-
-/// Loads a UIImage from the embedded Flutter project's assets.
```

Comment:
oh snap, you did the reverse too. That's awesome. Definitely make it clearer in the README.md

Reply:
added example in the readme